pkg/scaffold/api.go

```diff
@@ -52,6 +53,9 @@ type API struct {
 
 	// Force indicates that the resource should be created even if it already exists.
 	Force bool
+
+	// Indicates that the Resource structure allows support to multigroups
+	Multigroup bool
```

Comment:
I think you are not using this field for anything, as the `Multigroup` is stored in `project.Multigroup` and then it is injecting it in each file.